### PR TITLE
{CI} Add ccoa announcement

### PR DESCRIPTION
**Pull request by wangzelin007**
image

**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: .github/workflows/CCOA.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          else
+            echo "continue=false" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Add do-not-merge label
+        if: steps.date_check.outputs.continue == 'true'
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          labels: do-not-merge
```
Comment (MoChilia) [inline .github/workflows/CCOA.yml:38]: `actions-ecosystem/action-add-labels` is a third-party action, how about using `github.rest.issues.addLabels` in `actions/github-script`?

File: .github/workflows/CCOA.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+name: CCOA
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+
```
Comment (AllyW) [inline .github/workflows/CCOA.yml:8]: branches:
      - dev

**Review by MoChilia — APPROVED** [review]

File: .github/workflows/CCOA.yml (hunk 2 above)
Comment (AllyW) [reply .github/workflows/CCOA.yml:8]: suggest adding to avoid potential security mitigation omission

**Review by AllyW — DISMISSED** [review]

**Review by MoChilia — APPROVED** [review]

**Review by jsntcy — CHANGES_REQUESTED** [review]
Main repo doesn't need add this warning message as pull requests in main repo are allowed to be merged.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**